citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -135,7 +131,7 @@ protected void handleExportRequest(HttpServletRequest req, HttpServletResponse r
 			Reference ref, String format, String subtype) 
 			throws EntityNotDefinedException, EntityAccessOverloadException, EntityPermissionException 
 	{
-		if(! ContentHostingService.allowGetResource(req.getParameter("resourceId")))
+		if(! ContentHostingService.allowGetResource(req.getParameter("contentCollectionId")))
```

Comment:
This looks similar to a fix Surya was developing is this aiming to fix a permissions issue?